File: backend/internal/models/tombola.go
package models

// Participant représente un participant à un événement
type Tombola struct {
	ID         uint `gorm:"primaryKey" json:"id"`
	ActorID    uint `json:"actor_id" gorm:"not null"`
	KermesseID uint `json:"kermesse_id" gorm:"not null"`
}

// Participant représente un participant à un événement
type TombolaRequest struct {
	ID         uint `gorm:"primaryKey" json:"id"`
	ActorID    uint `json:"actor_id" gorm:"not null"`
	KermesseID uint `json:"kermesse_id" gorm:"not null"`
	NbTicket   int  `json:"nb_ticket" gorm:"not null"`
}

File: client/lib/core/models/tombola.dart
class Tombola {
  final int id;
  final int actorId;
  final int kermesseId;
  final int nbTicket;

  Tombola({
    required this.id,
    required this.actorId,
    required this.kermesseId,
    required this.nbTicket,
  });

  factory Tombola.fromJson(Map<String, dynamic> json) {
    return Tombola(
      id: json['id'],
      actorId: json['actor_id'],
      kermesseId: json['kermesse_id'],
      nbTicket: json['nb_ticket'],
    );
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'actor_id': actorId,
      'kermesse_id': kermesseId,
      'nb_ticket': nbTicket,
    };
  }
}